data/sql/updates/pending_db_world/rev_1752706389703869030.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Bone Witch
+UPDATE `creature_template` SET `minlevel` = 71, `maxlevel` = 71 WHERE (`entry` = 16380);
+
+UPDATE `quest_template_addon` SET `RewardMailDelay`=2*24*3600 WHERE `ID` IN (9295, 9299, 9300, 9301, 9302, 9304);
+
+-- Delete rare loot (lvl 60) and add epic loot (copied from Bone Witch - 16380)
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 16143) AND `Item` IN (23085, 23087, 22484, 23090, 23091, 23092, 23093);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```

Comment:
I know they came from another creature, but I really am not a fan of these drop chances.